File: data-prep/dataprep-util/src/main/scala/de/hpi/isg/dataprep/cases/Language.scala
package de.hpi.isg.dataprep.cases

/**
  *
  * @author Lan Jiang
  * @since 2018/8/28
  */
abstract class Language

case class EN() extends Language

case class DE() extends Language

object Language {

  def getLanguage(language: Language): String = {
    language match {
      case EN() => "English"
      case DE() => "Deutsch"
    }
  }
}

File: data-prep/dataprep-util/src/main/java/de/hpi/isg/dataprep/util/Language.java
package de.hpi.isg.dataprep.util;

import java.io.Serializable;

/**
 * @author lan.jiang
 * @since 1/26/19
 */
public enum Language {
    EN("English"),
    DE("Deutsch");

    private String language;

    Language(String language) {
        this.language = language;
    }

    public String getLanguage() {
        return language;
    }
}
